pkg/actor/system_test.go

```diff
@@ -220,7 +220,7 @@ func TestActorSendReceive(t *testing.T) {
 		t.Fatal("Timed out")
 	}
 
-	wait(t, time.Second, func() {
```

Comment:
Please add some comments, otherwise the magic number makes it hard to maintain in the feature.